### ISSUE TITLE
RC 2 docker upgrade from 10.6.4 fails to start

### ISSUE THREAD

**Issue by SteveDinn** [jellyfin/jellyfin#4913]
**Describe the bug**
Pulled the latest `jellyfin/jellyfin:stable-rc`.
Copied over my Jellyfin config and cache folders to jellyfin-rc folder.
Started new container based on RC image, but it failed to start up properly.

**System (please complete the following information):**
 - OS: Ubuntu 20.04 on RPi 4
 - Virtualization: docker
 - Clients: n/a
 - Browser: n/a
 - Jellyfin Version: 10.7 rc2
 - Playback: n/a
 - Installed Plugins: none
 - Reverse Proxy: n/a
 - Base URL: none
 - Networking: host
 - Storage: local

**To Reproduce**

- have working 10.6.4 Jellyfin instance
- have config and cache volume mount points in jellyfin folder
- `mkdir jellyfin-rc`
- `rsync -a -v --delete jellyfin/ jellyfin-rc/`
- copy docker compose section but replace config and cache volume mount points with jellyfin-rc
- `docker stop jellyfin`
- `docker-compose up jellyfin-rc`
- get following error (see log snippet)

**Expected behavior**

Jellyfin should start up normally.

**Logs**

```
jellyfin-rc          | [10:49:34] [INF] [1] Main: Operating system: Linux
jellyfin-rc          | [10:49:34] [INF] [1] Main: Architecture: Arm64
jellyfin-rc          | [10:49:34] [INF] [1] Main: 64-Bit Process: True
jellyfin-rc          | [10:49:34] [INF] [1] Main: User Interactive: True
jellyfin-rc          | [10:49:34] [INF] [1] Main: Processor count: 4
jellyfin-rc          | [10:49:34] [INF] [1] Main: Program data path: /config
jellyfin-rc          | [10:49:34] [INF] [1] Main: Web resources path: /jellyfin/jellyfin-web
jellyfin-rc          | [10:49:34] [INF] [1] Main: Application directory: /jellyfin/
jellyfin-rc          | [10:49:35] [INF] [1] Emby.Server.Implementations.AppBase.BaseConfigurationManager: Setting cache path: /cache
jellyfin-rc          | [10:49:35] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Defined LAN addresses : [10.1.1.0/24,127.0.0.1/8]
jellyfin-rc          | [10:49:35] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Defined LAN exclusions : []
jellyfin-rc          | [10:49:35] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Using LAN addresses: [10.1.1.0/24,127.0.0.1/8]
jellyfin-rc          | [10:49:35] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Using bind addresses: []
jellyfin-rc          | [10:49:35] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Using bind exclusions: []
jellyfin-rc          | [10:49:35] [INF] [1] Emby.Server.Implementations.ApplicationHost: Loading assemblies
jellyfin-rc          | [10:49:35] [FTL] [1] Main: Error while starting server.
jellyfin-rc          | System.TypeLoadException: Could not load type 'MediaBrowser.Model.Services.IService' from assembly 'MediaBrowser.Model, Version=10.7.0.0, Culture=neutral, PublicKeyToken=null'.
jellyfin-rc          |    at System.Reflection.RuntimeAssembly.GetExportedTypes()
jellyfin-rc          |    at Emby.Server.Implementations.Plugins.PluginManager.LoadAssemblies()+MoveNext()
jellyfin-rc          |    at Emby.Server.Implementations.ApplicationHost.GetComposablePartAssemblies()+MoveNext()
jellyfin-rc          |    at Emby.Server.Implementations.ApplicationHost.GetTypes(IEnumerable`1 assemblies)+MoveNext()
jellyfin-rc          |    at System.Collections.Generic.LargeArrayBuilder`1.AddRange(IEnumerable`1 items)
jellyfin-rc          |    at System.Collections.Generic.EnumerableHelpers.ToArray[T](IEnumerable`1 source)
jellyfin-rc          |    at System.Linq.Enumerable.ToArray[TSource](IEnumerable`1 source)
jellyfin-rc          |    at Emby.Server.Implementations.ApplicationHost.DiscoverTypes()
jellyfin-rc          |    at Emby.Server.Implementations.ApplicationHost.Init()
jellyfin-rc          |    at Jellyfin.Server.Program.StartApp(StartupOptions options)
jellyfin-rc          | [10:49:35] [INF] [1] Emby.Server.Implementations.ApplicationHost: Disposing CoreAppHost
jellyfin-rc          | [10:49:35] [INF] [2] Main: Received a SIGTERM signal, shutting down
```

**Comment by TheNamelessWonderer** [comment]
I got this error too. Delete all the plugins stored under `.local/share/jellyfin/plugins` (not sure about docker) and restarting Jellyfin solves it. You can then reinstall all your plugins again

**Comment by SteveDinn** [comment]
Awesome! I thought I had no plugins, but I forgot I installed trackt. Seems to be working now.

Thanks!

**Comment by TheNamelessWonderer** [comment]
I might be wrong but this PR might solve the issue: https://github.com/jellyfin/jellyfin/pull/4910

**Comment by sleach** [comment]
I fixed it by removing plugins and restarting, but as soon as I re-enable the plugins (InfuseSync and Trakt) and restart again it gets the same error. So at this point I can't use plugins so I have them disabled (I am on the docker version)

**Comment by TheNamelessWonderer** [comment]
Trakt has been updated for 10.7 but are you sure InfuseSync has? I can't find any information about it being updated for 10.7. The last update was in Oct 2, 2020.

**Comment by crobibero** [comment]
InfuseSync has a prerelease update https://github.com/firecore/InfuseSync/releases/tag/v1.3.1

**Comment by TheNamelessWonderer** [comment]
I see unfortunately that isn't in the Jellyfin Plugin Repo yet so would have to be installed manually.

**Comment by sleach** [comment]
Yep you all were right. Disabling InfuseSync for now fixed it. Trakt plugin works fine. Thanks

**Comment by Oatavandi** [comment]
I did a clean install of 10.7.0 RC-2 for docker

Error:
standard_init_linux.go:222: exec user process caused "no such file or directory"

Same error, while trying the latest unstable
but works fine with latest:stable 10.6.4
Also working fine with  jellyfin:10.7.0-rc1

Even it was working fine with unstable 2 days before

**Comment by itshaadi** [comment]
the exact same error happens on `10.7.0 RC3` and it was caused by the `IPTV_6.0.0.0` plugin.

**Comment by nielsvanvelzen** [comment]
Should be fixed in the next RC by #5090

**Comment by Akirainblack** [comment]
Still occurs with RC4
